client/js/editmode.js

```diff
@@ -491,6 +491,11 @@ async function onClickUpdateWidget(applyChangesFromUI) {
       return;
     }
 
+    if(widget.parent !== undefined && (widget.parent === null || !widgets.has(widget.parent))) {
```

Comment:
I believe test should be:
`(widget.parent !== undefined && !(widget.parent === null || widgets.has(widget.parent)))`
otherwise putting `"parent":null` in the JSON would prevent the update.

Reply:
It shouldn't because the JSON editor deletes every property that equals its default.

Reply:
I understand that the JSON editor removes `null`, but for default properties, it displays the default value of `null`. The "basic" editor should accept `null` (possibly removing it before saving to state - either with or without informing the user).